source/clear-linux/get-started/virtual-machine-install/vmw-player.rst

```diff
@@ -65,18 +65,18 @@ For additional help, see the `VMware Workstation Player Documentation`_.
 Download the latest |CL| installer ISO
 **************************************
 
-Get the latest |CL| installer ISO image from the `image`_ repository.
-Look for :file:`clear-[version number]-installer.iso.xz`.
+Get the latest |CL| installer ISO image from the `download`_ repository.
+Look for :file:`clear-[version number]-live-desktop.iso.xz`.
```

Comment:
Please simplify this. Something like: 
"Visit our downloads page to get the latest installer with Clear Linux OS Desktop." 
(The previous version is a holdover from when we didn't have the Downloads page.) 
I presume you prefer to use the GUI installer here.

Reply:
Updated with latest commit.